README.md

```diff
@@ -93,41 +118,21 @@ Before you start working you will need to install the dependencies for the clien
 1. Move into the `client` directory (`cd client`)
 2. Run `npm install`
 
-### Enable ESLint in VS Code
-
-ESLint is a tool for checking the quality and style of your client TypeScript
-and template HTML code, and can provide valuable "live" feedback on
-your coding in VS Code.
-
-Since this is the first time we will be using ESLint there is an additional step to make sure the VS Code extension is working in the project. When you first open a TypeScript file you will see at the bottom right that ESLint is disabled.
-
-![image](https://user-images.githubusercontent.com/1300395/107999308-bc59ec80-6fac-11eb-9784-75a471a50aa4.png)
-
-Click the red "ESLINT" to open this dialog:
-
-![image](https://user-images.githubusercontent.com/1300395/107996971-528b1400-6fa7-11eb-89bc-afc71747f820.png)
-
-Click "Allow Everywhere" to enable ESLint.
-
-You can also open this dialog with the following steps:
-
-1. Hit `CTRL + SHIFT + P` (`⌘ + ⇧ + P` on Macs) to open the Command Palette. You can also find this by going to the "View" menu and clicking "Command Palette..."
-2. Start typing and select "ESlint: Manage Library Execution". That should open a dialog seen above.
-
 ### Seeding the Database
 
-To give yourself some data to work with instead of starting with an empty database in our development environment, you need to 'seed' the database with some starter data. Seed data and the seed script are stored in the top level directory `database`. To seed the database, move into that directory and run `./mongoseed.sh` (or `.\mongoseed.bat` on Windows). This will take each of the JSON files in `database/seed/` and insert their elements into the `dev` database.
```

Comment:
I think we might as well leave the windows command, we are leaving in the bat file and there's a chance a student will want to do things on their personal machine still.